server/utils/nunjucksSetup.ts

```diff
@@ -29,6 +29,10 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
     })
   }
 
+  registerNunjucks(app)
```

Comment:
This diff is hard to read, but basically I had to refactor the `nunkucksSetup` file a little.
Previously it had 1 function in it that did everything in respect of setting up the nunjucks env.

The new test that tests the logic in the nunjucks view needs a nunjucks environment spun up (in order for it to know how to process nunjucks templates and constructs etc), but it doesnt need everything that this function did

So I split it into 2 functions, where the first (original function) calls the 2nd (new) function . The 2nd (new) function has everything that was moved out of the first function, and is also called by the test.

See test below 👇